pages/download.py

```diff
@@ -299,6 +312,10 @@ def get_content(self):
 """) + "\n\n" + """
 [url_addfriend]: http://127.0.0.1:8888/addfriend/
 [url_seednode_info]: https://wiki.freenetproject.org/Seed_nodes#Seed_node
+""") + "\n\n" + _("""
+You don't have to add friends right now. Freenet will automatically connect to strangers
+and will work just fine. However, your (or someone elses) government may be able to find out who you are with
```

Comment:
Without friends, Freenet will only work "just fine" if low or medium security is chosen.